src/widget/wtrackmenu.cpp

```diff
@@ -253,7 +255,10 @@ void WTrackMenu::createActions() {
         // The keypress is caught in WTrackTableView::keyPressEvent
         if (m_pTrackModel) {
             m_pPropertiesAct->setShortcut(
-                    QKeySequence(kPropertiesShortcutModifier | kPropertiesShortcutKey));
+                    // TODO(XXX): Qt6 replace enum | with QKeyCombination
+                    QKeySequence(
+                            static_cast<Qt::Key>(kPropertiesShortcutModifier) |
+                            kPropertiesShortcutKey));
```

Comment:
`QKeySequence::QKeySequence(int k1, int k2 = 0, int k3 = 0, int k4 = 0)`
Being pedantic, it is wrong to cast a Qt::Modifier to a Qt::Key. The QKeySequence expects an int. 
Does this work? 
```
QKeySequence(
                            static_cast<int>(kPropertiesShortcutModifier) |
                            kPropertiesShortcutKey));
```